examples/response_stream/public/containers/app/pages/page_simple_string_stream/index.tsx

```diff
@@ -34,7 +34,7 @@ export const PageSimpleStringStream: FC = () => {
   const { dispatch, errors, start, cancel, data, isRunning } = useFetchStream<
     ApiSimpleStringStream,
     typeof basePath
-  >(`${basePath}/internal/response_stream/simple_string_stream`, '1', {
+  >(core.http, `${basePath}/internal/response_stream/simple_string_stream`, '1', {
```

Comment:
When I run Kibana with a base path, the `log_rate_analysis` analysis is returning a `404` error. Looks like the base path is being duplicated:

<img width="650" alt="image" src="https://github.com/elastic/kibana/assets/7405507/41ec8e3a-8d72-46d0-9cb6-3b7214f7ddf9">

Without a base path, all works fine.

Reply:
Thanks for catching this, I missed it. Previously with plain `fetch` we had to add the base path ourselves, now Kibana's `http` service takes care of it. Fixed in 0492f82.